DOCS-7334 API Groups (#106)
mulesoft/docs-exchange: modules/ROOT/pages/about-sharing-assets.adoc

endif::[]
:imagesdir: ../assets/images/

> You can use Exchange private or Exchange portal to share Mule XML example code, templates, connectors, policies, APIs, API Groups, and custom information assets. Sharing assets helps your organization model and reuse code and APIs for ease of access and to provide a repository.

Reviewer: Is "Exchange private" a thing?
Do we mean that you can use Exchange locally or via a public portal?

Reviewer: ```suggestion
You can use Exchange private or Exchange portal to share Mule XML example code, templates, connectors, policies, APIs, API Groups, and custom information assets. Sharing assets helps your organization model and reuse code and APIs to simplify access and to provide a repository.
```

Author: Fixed.

> Exchange private makes it easy to share assets with internal developers by giving them viewer, contributor, or administrator access. Additionally, assets created within a business group are automatically shared with developers within that business group.

Reviewer: ditto re. "Exchange private"?

Author: Fixed.
